Twittimation/Commands/Tweet.cs

```diff
@@ -0,0 +1,71 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using Twittimation.IO;
+
+namespace Twittimation.Commands
+{
+    public class Tweet : Command
+    {
+        public override string Name { get; } = "Tweet";
+        public override List<string> RequiredArgs { get; } = new List<string>() { "Text" };
+        public override List<string> OptionalArgs { get; } = new List<string>();
+        public override Optional<string> OptionalRepeatedArg { get; } = new Optional<string>();
+        public override string HelpInfo { get; } = "Posts a tweet with the specified text.";
+        public override string ExtendedHelp { get; } = "Posts a tweet with the specified text using the saved credentials.";
+
+        private AppDataJsonIo _io;
+
+        public Tweet(AppDataJsonIo io)
+        {
+            _io = io;
+        }
+
+        public override void Go(string[] args)
+        {
+            ValidateArgCount(args);
+            if (!_io.HasSave("Credentials"))
+                Console.Error.WriteLine("No credentials were provided for this request");
```

Comment:
Should end the command if no credentials are present.

Reply:
Good catch, will fix.